src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -267,7 +295,8 @@ public void AddPeer(ITxPoolPeer peer)
         {
             if (_broadcaster.AddPeer(peer))
             {
-                _broadcaster.BroadcastOnce(peer, _transactionSnapshot ??= _transactions.GetSnapshot());
+                _broadcaster.AnnounceOnce(peer, _transactionSnapshot ??= _transactions.GetSnapshot());
+                _broadcaster.AnnounceOnce(peer, _blobTransactionSnapshot ??= _blobTransactions.GetSnapshot());
```

Comment:
Can it be one message?

Reply:
It would be better, but it is not easy to implement it well. We would need to:
1) concat collections. We will have one message, but we will introduce a lot of unnecessary allocations
or 2) send 2 collections further. But to iterate through them separately, it will need to be passed as far as up to `SyncPeerProtocolHandlerBase` and all overrides. A lot of refactorings.

This part will be changed anyway, to not announce txs to peers with way bigger head block number (to avoid announcing already included txs when we are out of sync). Let's leave it as it is for now and I will try to figure something out when implementing described feature

And another change will be introduced when implementing eth/69, then probably we will need to iterate it in TxBroadcaster, so 2 messages problem will be solved as well

Reply:
Notify in the end takes `IEnumerable`, so `AnnounceOnce` could take it to - and we can pass a lazy one without allocations.